service/camera-streamer-pi219-8MP.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=camera-streamer web camera
+After=network.target
+
+[Service]
+ExecStart=/usr/local/bin/camera-streamer \
+  -camera-path=/base/soc/i2c0mux/i2c@1/imx219@10 \
+  -camera-type=libcamera \
+  -camera-format=YUYV \
+  -camera-width=3280 -camera-height=2464 \
+  -camera-fps=30 \
+  ; use two memory buffers to optimise usage
+  -camera-nbufs=2 \
+  ; the high-res is 1640x1232
+  -camera-high_res_factor=2 \
+  ; the low-res is 820x616
+  -camera-low_res_factor=4 \
+  ; bump brightness slightly
+  -camera-options=brightness=0.1 \
+  ; disable auto-focus
```

Comment:
This can be removed I think, as this camera does not have `auto-focus`.

Reply:
Done!